chainntnfs/height_hint_cache.go

```diff
@@ -127,6 +127,10 @@ func (c *HeightHintCache) CommitSpendHint(height uint32, ops ...wire.OutPoint) e
 		return nil
 	}
 
+	if len(ops) == 0 {
```

Comment:
It's possible in the event that we don't need to update the hints for any transactions/outpoints. In that case, rather than handling the check there, I figured it'd be better to have it within the cache itself to avoid the extra database hit.